go.mod

```diff
@@ -269,6 +269,7 @@ require (
 	go.starlark.net v0.0.0-20200306205701-8dd3e2ee1dd5 // indirect
 	go.uber.org/multierr v1.8.0 // indirect
 	go4.org/intern v0.0.0-20220617035311-6925f38cc365 // indirect
+	go4.org/netipx v0.0.0-20220812043211-3cc044ffd68d // indirect
```

Comment:
should it be direct dependency?

Reply:
Rerun `go mod tidy` - should be good now.